Dockerfile

```diff
@@ -28,7 +28,7 @@ RUN confluent-hub install --no-prompt cjmatta/kafka-connect-sse:latest
 RUN confluent-hub install --no-prompt jcustenborder/kafka-connect-json-schema:latest
 
 # Install Elasticsearch connector
-RUN confluent-hub install --no-prompt confluentinc/kafka-connect-elasticsearch:10.0.2
+RUN confluent-hub install --no-prompt confluentinc/kafka-connect-elasticsearch:latest
```

Comment:
Checking - do we intend to re-pin this later, or let if float?  If re-pin, might be worth adding a TODO comment.

Reply:
@javabrett -- PR is now updated to pin to 11.0.0 (cc: @rspurgeon )